backend/tests/index.test.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * @jest-environment node
+ */
+const request = require("supertest");
+const app = require("../app");
+const DataBase = require("../database");
+const DB = new DataBase();
+const expectedJSON = {
+  originalUrl: "http://www.github.com",
+  shorturlId: "vnFthnWBJ",
+  creationDate: "2021-03-07 10:39:11",
+  redirectCount: 0,
+};
+const expectedURL = {
+  url: "http://www.github.com",
+};
+const notRealURL = {
+  url: "www.facebokde.com",
+};
+
+beforeEach(async () => {
+  await DB.putBin([expectedJSON]);
+});
+
+describe("POST route", () => {
+  it("Should return the same shortURL every time", async () => {
+    const response = await request(app)
+      .post("/api/shorturl/new/")
+      .send(expectedURL);
+    expect.assertions(2);
+    // Is the status code 200
+    expect(response.status).toBe(200);
+
+    // Is Json created properly
+    expect(response.body).toEqual(expectedJSON);
+  });
+
+  it("Should return error", async () => {
+    const response = await request(app)
+      .post("/api/shorturl/new/")
+      .send(notRealURL);
+    expect.assertions(2);
+    // Is the status code 401
+    expect(response.status).toBe(401);
+
+    // Is the error massage the same
+    expect(response.body).toEqual("URL is not valid");
+  });
+});
+
+describe("Statistic route", () => {
```

Comment:
Missing a check for `404` in `statistic` route

Reply:
Please check lines 60-66, it's in there.

Reply:
oops, I meant `400`